ogre2/src/Ogre2DepthCamera.cc

```diff
@@ -283,6 +283,32 @@ void Ogre2DepthCamera::CreateDepthTexture()
   psParams->setNamedConstant("tolerance",
       static_cast<float>(1e-6));
 
+  // create background material is specified
+  MaterialPtr backgroundMaterial = this->Scene()->BackgroundMaterial();
+  bool validBackground = backgroundMaterial &&
+      !backgroundMaterial->EnvironmentMap().empty();
+
+  if (validBackground)
+  {
+    Ogre::MaterialManager &matManager = Ogre::MaterialManager::getSingleton();
+    std::string skyMatName = "SkyBox_" + this->Name();
```

Comment:
The `SkyBox` string is repeated in a few places. How about storing it somewhere as a constant?

Reply:
done in 0823885 and fbae46078e563bbda29f6dbfd713f4cd4e976919

Reply:
Thanks. Does the `SkyBox` string in `RenderTarget` need to match the one in `DepthCamera`? Keeping both in the same place could help future readers understanding how everything fits together, and prevent errors in case one is changed but not the other.

I'll leave it up to you if you think it's worth it.